docs/docs/writing-a-long-lived-client.md

```diff
@@ -44,11 +39,11 @@ if (providedConstraint != null && loadedConstraint != providedConstraint) {
     );
 }
     
-// TODO client interacts with enclave.
+// Code where the client interacts with the enclave.
 
 Files.write(stateFile, enclaveClient.save())
 ```
 
-Notice how an [enclave constraint](constraints.md) is not required when loading the state from the file.
-In general, we assume that the constraint will not change for a long-lived client. The constraint can always be
-updated using [`EnclaveClient.setEnclaveConstraint`](api/-conclave%20-core/com.r3.conclave.client/-enclave-client/set-enclave-constraint.html).
+Note that you don't need an [enclave constraint](constraints.md) to load the state from the file. This is because the
```

Comment:
I am not really sure whether this paragraph is relevant for a first time reader of this tutorial

Reply:
Removed.